src/components/UIShell/UIShell.js

```diff
@@ -228,7 +228,7 @@ function UIShell({
         skipToContentProps={skipToContentProps}
         requestSummary={user.requestSummary}
         notificationsConfig={{
-          wsUrl: `${finalBaseServiceUrl}/notifications/ws`,
+          wsUrl: `${finalBaseServiceUrl}/notifications/ws`.replace("https://", "wss://"),
```

Comment:
Meh, but I'm not sure of a better way to do this without requesting service changes